package.json

```diff
@@ -20,46 +20,46 @@
     "Twitter, Inc."
   ],
   "scripts": {
-    "start": "npm-run-all --parallel watch docs-serve",
+    "start": "npm-run-all --aggregate-output --parallel watch docs-serve",
```

Comment:
Mh this prevents the output of long living tasks like docs-serve 🤔

Reply:
Good catch! I'll revert this one.